jukebox_radio/streams/views/queue_interval/create_view.py

```diff
@@ -11,24 +11,25 @@ def post(self, request, **kwargs):
         """
         QueueInterval = apps.get_model("streams", "QueueInterval")
 
-        queue_uuid = request.POST.get("queueUuid")
-
-        lower_bound_marker_uuid = request.POST.get("lowerBoundMarkerUuid")
-        upper_bound_marker_uuid = request.POST.get("upperBoundMarkerUuid")
-
-        is_muted = request.POST.get("isMuted") == "true"
-        repeat_count = request.POST.get("repeatCount")
-        repeat_count = repeat_count if repeat_count else None
-
+        queue_uuid = self.param(request, "queueUuid")
+        lower_bound_marker_uuid = self.param(request, "lowerBoundMarkerUuid")
```

Comment:
I made `self.params` for this case. the front-end sends the string `"null"`. This is some wonky request middleware to remedy the issue.

I know elsewhere in the codebase there is an issue with Django reading `"true"` from the front-end instead of `True`, so later the `self.params` method will be updated to include catching boolean cases as well.